pkg/config/config_unsupported.go

```diff
@@ -0,0 +1,7 @@
+// +build !linux
```

Comment:
Do we actually need these files.  selinux pkg should work fine on non selinux supported systems?

Reply:
I deleted this but I saw this file is used in cri-o.

Reply:
@rhatdan  the darwin CI failed, still need to add this config_unsupported.go to containers/common?